README.md

```diff
@@ -1,2 +1,7 @@
-# bats-library
-Bash Automated Testing System Library
+# 🦇BATS Library 🦇
+[Bash Automated Testing System](https://github.com/bats-core/bats-core) Library - a collection of `_helper` functions and files for use in conjunction with `conftests` and the CoP's Rego Policies.
```

Comment:
the only comment is to remove the conftest/rego bits. i'd hope this lib can be used by multiple things, i.e.:
- https://github.com/redhat-cop/containers-quickstarts/blob/master/rabbitmq/test/acceptance/_helpers.bash

which is not related to conftest/rego